.github/workflows/ci.yml

```
@@ -24,7 +24,9 @@ jobs:
         cache: 'npm'
     
     - name: Install dependencies
-      run: npm ci
+      run: |
+        npm cache clean --force
+        npm install
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)